pinot-common/src/main/java/org/apache/pinot/common/datablock/BaseDataBlock.java

```diff
@@ -71,9 +71,6 @@
  * +-----------------------------------------------+
  * | VARIABLE_SIZE_DATA SECTION                    |
  * +-----------------------------------------------+
- * | METADATA LENGTH                               |
- * | METADATA SECTION                              |
- * +-----------------------------------------------+
```

Comment:
This wasn't useful. All blocks but metadata blocks left this empty and in metadata blocks the content was equivalent to variable_size_data, but codified as a single string.